vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java

```diff
@@ -0,0 +1,3 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
????

Reply:
Hmm, what's wrong with this one? Thought it was the most recent? >_< IntelliJ uses that as a template for me, so I don't really notice when these appear.

Reply:
I was wondering if this class was added intentionally or not

Reply:
I probably looked at the code prior to force push.